x-pack/plugin/sql/qa/server/src/main/resources/datetime.csv-spec

```diff
@@ -1570,3 +1570,16 @@ SELECT CAST (CAST (birth_date AS VARCHAR) AS TIMESTAMP) a FROM test_emp WHERE YE
 ---------------
 1965-01-03T00:00:00Z
 ;
+
+// datetime in range
+dateInRangeFilter
+select hire_date from test_emp where hire_date > 'now-100y' and hire_date < 'now+2y' limit 5;
```

Comment:
Does the other query `select hire_date from test_emp where hire_date > 'now-25y' and hire_date < '2021-09-01'` mentioned in the issue work as well?

Reply:
Yes, I have tested  it  on my local cluster using this branch. 
![image](https://user-images.githubusercontent.com/30148291/134907793-22833695-4b34-4c7f-b83b-d0f7bfa6c460.png)

I use `hire_date > 'now-100y' and hire_date < 'now+2y'  ` in my test case  because I think it's more representative (of course , which also return empty result in the original code because of  `areBoundariesInvalid()`)

Reply:
👍 I guess it makes sense to have both cases (or even more) in the specs.

Reply:
OK, I have add more test cases (such as the query `select hire_date from test_emp where hire_date > 'now-25y' and hire_date < '2021-09-01'` mentioned in the issue ) in the specs,  and pass in my local cluster

Reply:
> +1 I guess it makes sense to have both cases (or even more) in the specs.

OK, I have add more test cases (such as the query select hire_date from test_emp where hire_date > 'now-25y' and hire_date < '2021-09-01' mentioned in the issue ) in the specs, and pass in my local cluster